### PR TITLE
Feature/new dashboard bridge

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Bridge

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/constants/existingTokens.js [hunk 1]
```diff
@@ -1,62 +1,50 @@
 import DaiIcon from 'images/dai_logo.png'
 import UsdcIcon from 'images/usdc_logo.png'
-import TrustIcon from 'images/trust_token_logo.png'
 import TetherIcon from 'images/tether.png'
-import EursTokenIcon from 'images/eurs_token.png'
-import DigitalRandIcon from 'images/digital_rand_logo.png'
-import RupiahIcon from 'images/rupiah_logo.png'
+import MoneriumIcon from 'images/monerium.jpeg'
+// import DigitalRandIcon from 'images/digital_rand_logo.png'
```
Comment (leonprou) [inline dapp/src/constants/existingTokens.js:5]: Can you delete the comments?

File: dapp/src/components/common/withBridge/index.jsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+          bridge === 'foreign'
+            ? CONFIG.web3.bridge.confirmations.foreign
+            : CONFIG.web3.bridge.confirmations.home
+        )
+        promise.on('confirmation', (confirmationNumber) => {
+          setConfirmationNumber(confirmationNumber)
+        })
+
+        const _web3Home = getWeb3({ networkType: homeNetwork })
+        const _web3Foreign = getWeb3({ networkType: foreignNetwork })
```
Comment (leonprou) [inline dapp/src/components/common/withBridge/index.jsx:87]: Can you use web3 instances from the dashboard store here?

File: dapp/src/store/mobx/state/dashboard.js [hunk 3]
```diff
@@ -247,15 +396,17 @@ export default class Dashboard {
     }
   })
 
-  checkIsCommunityMember = flow(function * (accountAddress) {
```
Comment (leonprou) [inline dapp/src/store/mobx/state/dashboard.js:250]: can you revert this?

**Review by leonprou — APPROVED** [review]

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good! see small comments

File: dapp/src/components/common/withBridge/index.jsx (hunk 2 above)
Comment (LiorAgnin) [reply dapp/src/components/common/withBridge/index.jsx:87]: 👍

File: dapp/src/constants/existingTokens.js (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/constants/existingTokens.js:5]: 👍

**Review by leonprou — APPROVED** [review]

**Pull request merged**